src/app/vehicle-technical-record/vehicle-technical-record.component.spec.ts

```diff
@@ -1,14 +1,21 @@
 import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { HttpClientTestingModule } from '@angular/common/http/testing';
 import { VehicleTechnicalRecordComponent } from './vehicle-technical-record.component';
+import { initialAppState } from '@store/.';
+import { provideMockStore } from '@ngrx/store/testing';
+import { TestRecordSummaryComponent } from '../features/test-record-summary/test-record-summary.component';
 
 describe('VehicleTechnicalRecordComponent', () => {
   let component: VehicleTechnicalRecordComponent;
   let fixture: ComponentFixture<VehicleTechnicalRecordComponent>;
 
   beforeEach(async () => {
     await TestBed.configureTestingModule({
-      declarations: [VehicleTechnicalRecordComponent]
-    }).compileComponents();
+      imports: [HttpClientTestingModule],
```

Comment:
Make linter normalise spacing within square brackets. Don't mind if space or not, just be consistent.